src/framework/lib/genesisframework/task.rb

```diff
@@ -124,7 +124,9 @@ def install provider, *what
                 || Gem::Dependency.new(item).matching_specs.count == 0
             end
             if gems.size > 0    # make sure we still have something to do
-              Kernel.system('gem', 'install', '--no-ri', '--no-rdoc', *gems)
+              options = ['--no-ri', '--no-rdoc', config.fetch(:gems_source, '').split]
```

Comment:
In our setup we have something along the lines of `:gems_source: --clear-sources --source https://internal-gemserver.com` in the genesis config.

I'll add a `:gem_source:` to the example config in the testenv with some documentation.